optimum/habana/transformers/models/llama/modeling_llama.py

```diff
@@ -638,20 +643,10 @@ def forward(
         if inputs_embeds is None:
             inputs_embeds = self.embed_tokens(input_ids)
 
-        if self._use_sdpa and not output_attentions:
```

Comment:
instead of removal, can we have similar change in falcon https://github.com/huggingface/optimum-habana/blob/main/optimum/habana/transformers/models/falcon/modeling_falcon.py#L450 for easier code maintenance?